package.json

```diff
@@ -12,7 +12,7 @@
     "build:web": "expo build:web",
     "build:ios": "expo build:ios",
     "build:android": "expo build:android",
-    "build:local:ios": "turtle build:ios --type=simulator --output=build/polaris.tgz && tar xzf build/polaris.tgz -C build",
```

Comment:
this looks good because you are providing these CLI args in https://github.com/nearform/polaris/pull/233/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R93. The only thing I don't like is it introduces an inconsistency between the android script, which provides them, and the ios one, which now doesn't. I would try to keep them conceptually similar, maybe removing the CLI args from the android script and providing them inside the workflow file instead, as you're doing with the ios one.

Reply:
👍 Didn't touch the prev code, but makes sense to be consistent. I'll merge like this and will fix after this is merged to master. If in the future someone wants to build locally they'll just have to update this...